File: python/task2b/S2_FOR_05.py
# S2_FOR_05 - Multiplication table 10x10

print("10x10 Multiplication Table:\n")

# Header
header = "    "
for j in range(1, 11):
    header += f"{j:4}"
print(header)
print("-" * 45)

# Rows
for i in range(1, 11):
    row = f"{i:2} |"
    for j in range(1, 11):
        row += f"{i * j:4}"
    print(row)

File: js/task2b/S2_FOR_05.js
// S2_FOR_05 - Multiplication table

console.log("10x10 Multiplication Table:\n");

// header row
let header = "    ";
for (let i = 1; i <= 10; i++) {
  header += String(i).padStart(4);
}
console.log(header);
console.log("-".repeat(44));

// table rows
for (let row = 1; row <= 10; row++) {
  let line = String(row).padStart(2) + " |";
  for (let col = 1; col <= 10; col++) {
    line += String(row * col).padStart(4);
  }
  console.log(line);
}
